shell/browser/api/electron_api_web_contents.cc

```diff
@@ -3819,6 +3819,15 @@ gin::Handle<WebContents> WebContentsFromID(v8::Isolate* isolate, int32_t id) {
                   : gin::Handle<WebContents>();
 }
 
+gin::Handle<WebContents> WebContentsFromDevToolsTargetID(
+    v8::Isolate* isolate,
+    std::string target_id) {
+  auto agent_host = content::DevToolsAgentHost::GetForId(target_id);
+  return agent_host
+             ? WebContents::FromOrCreate(isolate, agent_host->GetWebContents())
```

Comment:
I don't think we want `OrCreate` here. If there's an existing WebContents, we should return it, but it would be surprising to _create_ one here.